<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qCjyxDiffusionTensorVolumeDisplayWidget</class>
 <widget class="qCjyxWidget" name="qCjyxDiffusionTensorVolumeDisplayWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>271</width>
    <height>121</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tensor Display Properties</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="ctkCollapsibleGroupBox" name="CollapsibleGroupBox_2">
     <property name="title">
      <string>Scalar Display</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="ScalarInvariantLabel">
        <property name="text">
         <string>Scalar Mode:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="qDMMLScalarInvariantComboBox" name="ScalarInvariantComboBox"/>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="qCjyxScalarVolumeDisplayWidget" name="ScalarVolumeDisplayWidget" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="ctkCollapsibleGroupBox" name="GlyphsOnSlicesDisplayCollapsibleGroupBox">
     <property name="title">
      <string>Glyphs on Slices Display</string>
     </property>
     <layout class="QFormLayout" name="formLayout_4">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="GlyphVisibilityLabel">
        <property name="text">
         <string>Slice Visibility:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QCheckBox" name="RedSliceCheckBox">
          <property name="text">
           <string>Red</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="YellowSliceCheckBox">
          <property name="text">
           <string>Yellow</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="GreenSliceCheckBox">
          <property name="text">
           <string>Green</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="qCjyxDTISliceDisplayWidget" name="DTISliceDisplayWidget" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>qDMMLScalarInvariantComboBox</class>
   <extends>QWidget</extends>
   <header>qDMMLScalarInvariantComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>qCjyxWidget</class>
   <extends>QWidget</extends>
   <header>qCjyxWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ctkCollapsibleGroupBox</class>
   <extends>QGroupBox</extends>
   <header>ctkCollapsibleGroupBox.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>qCjyxScalarVolumeDisplayWidget</class>
   <extends>QWidget</extends>
   <header>qCjyxScalarVolumeDisplayWidget.h</header>
   <container>1</container>
   <slots>
    <slot>setDMMLScene(vtkDMMLScene*)</slot>
   </slots>
  </customwidget>
  <customwidget>
   <class>qCjyxDTISliceDisplayWidget</class>
   <extends>QWidget</extends>
   <header>qCjyxDTISliceDisplayWidget.h</header>
   <container>1</container>
   <slots>
    <slot>setDMMLScene(vtkDMMLScene*)</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>qCjyxDiffusionTensorVolumeDisplayWidget</sender>
   <signal>dmmlSceneChanged(vtkDMMLScene*)</signal>
   <receiver>ScalarVolumeDisplayWidget</receiver>
   <slot>setDMMLScene(vtkDMMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>44</x>
     <y>66</y>
    </hint>
    <hint type="destinationlabel">
     <x>49</x>
     <y>61</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>qCjyxDiffusionTensorVolumeDisplayWidget</sender>
   <signal>dmmlSceneChanged(vtkDMMLScene*)</signal>
   <receiver>DTISliceDisplayWidget</receiver>
   <slot>setDMMLScene(vtkDMMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>231</x>
     <y>131</y>
    </hint>
    <hint type="destinationlabel">
     <x>236</x>
     <y>125</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
